dev/lint-python

```diff
@@ -61,13 +79,28 @@ else
 fi
 
 if [ "$lint_status" -ne 0 ]; then
-    echo "Python lint checks failed."
+    echo "PEP8 checks failed."
     cat "$PYTHON_LINT_REPORT_PATH"
 else
-    echo "Python lint checks passed."
+    echo "PEP8 checks passed."
 fi
 
-# rm "$PEP8_SCRIPT_PATH"
 rm "$PYTHON_LINT_REPORT_PATH"
 
+
+for to_be_checked in "$PATHS_TO_CHECK"
+do
+    pylint --rcfile="$PYLINT_HOME/pylintrc" $to_be_checked >> "$PYLINT_REPORT_PATH"
+done
+
+if [ "${PIPESTATUS[0]}" -ne 0 ]; then
+    lint_status=0
```

Comment:
Shouldn't this be assigning a non-zero value in order to trigger a build failure?